helix-core/src/main/java/org/apache/helix/manager/zk/CallbackHandler.java

```diff
@@ -818,4 +883,38 @@ public String getContent() {
         + ", _listener=" + _listener + ", _changeType=" + _changeType + ", _manager=" + _manager
         + ", _zkClient=" + _zkClient + '}';
   }
+
+  /**
+   * Used to shut down a periodic triggered task
+   */
+  private void shutDownTriggerTask(boolean isShutdown) {
+    if (_scheduledTriggerFuture != null) {
+      _scheduledTriggerFuture.cancel(true);
+    }
+    if (_periodicTriggerExecutor != null && isShutdown) {
+      _periodicTriggerExecutor.shutdownNow();
+      _periodicTriggerExecutor = null;
+    }
+  }
+
+  /**
+   * Used to initialize a periodic triggered task
+   * Schedule tasks in a task executor with fixed intervals
+   */
+  private void initTriggerTask() {
```

Comment:
nit, take _periodicTriggerInterval as the parameter, and don't refer to the global private field. This is a good style in which you can reduce the impact of a private field in the class. So less dependency if you want to make any further change.

Reply:
Great suggestion! Fixed. Thanks!